snowflake/pkg/version/version.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package version
```

Comment:
Shall we put this pkg under `pkg/` instead of `snowflake/pkg`?

Reply:
why? is it needed for other Theia code?

Reply:
BTW, given that `theia` and `theia/snowflake` are different Go modules, this is not straightforward and may not be a good idea.

Reply:
I was inspired by the version command you just added, our theia CLI doesn't have a version command so it might be useful to have one as well.
However, as you pointed out, it may not be advisable to do so across Go modules. Please disregard this comment.